plugins/grpc/objc/v1.64.2/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# syntax=docker/dockerfile:1.7
+FROM debian:bookworm-20240513 AS build
+
+ARG TARGETARCH
+
+RUN apt-get update \
+ && apt-get install -y curl git cmake build-essential autoconf clang libc++-dev libtool pkg-config unzip zip
+RUN arch=${TARGETARCH}; \
+    if [ "${arch}" = "amd64" ]; then arch="x86_64"; fi; \
+    curl -fsSL -o /usr/local/bin/bazel https://github.com/bazelbuild/bazel/releases/download/7.1.2/bazel-7.1.2-linux-${arch} \
```

Comment:
Hope bazel `7.2.0` doesn't give us issues. I guess we can update next time, although doesn't really make a difference.